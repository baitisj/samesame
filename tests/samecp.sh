NAME=samecp
PARAM=copy
. ${srcdir}/tests/include_action.sh
