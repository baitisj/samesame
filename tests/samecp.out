samecp -A:
./work/test1/file2a -> ./work/test1/file1a
./work/test1/file3a -> ./work/test1/file1a

samecp -Z:
./work/test1/file2a -> ./work/test1/file3a
./work/test1/file1a -> ./work/test1/file3a

samecp -At:
./work/test1/file3a -> ./work/test1/file1a
./work/test1/file2a -> ./work/test1/file1a

samecp -Zt:
./work/test1/file3a -> ./work/test1/file2a
./work/test1/file1a -> ./work/test1/file2a

samecp -L:
./work/test1/file1a -> ./work/test1/file2a
./work/test1/file3a -> ./work/test1/file2a
